Sources/Commands/SwiftTestTool.swift

```diff
@@ -198,6 +219,55 @@ public struct SwiftTestTool {
         let result: Void? = try? system(args, environment: NSProcessInfo.processInfo().environment)
         return result != nil
     }
+
+    /// Locates XCTestHelper tool inside the libexec directory and bin directory.
+    /// Note: It is a fatalError if we are not able to locate the tool.
+    ///
+    /// - Returns: Path to XCTestHelper tool.
+    private func xctestHelperPath() -> String {
+        let xctestHelperBin = "swiftpm-xctest-helper"
+        let binDirectory = Process.arguments.first!.abspath.parentDirectory
+        // XCTestHelper tool is installed in libexec.
+        let maybePath = Path.join(binDirectory, "../libexec/swift/pm/", xctestHelperBin)
+        if maybePath.isFile {
+            return maybePath
+        }
+        // This will be true during swiftpm developement.
+        let path = Path.join(binDirectory, xctestHelperBin)
```

Comment:
Can you drop a `FIXME` here that we need should factor all of the "development"-time resource location stuff into a common place?